test/sql/updates/setup.policies.sql

```diff
@@ -20,21 +20,32 @@ SELECT
 
 DO LANGUAGE PLPGSQL $$
 DECLARE
-  ts_version TEXT;
+  ts_major INT;
+  ts_minor INT;
 BEGIN
 
-  SELECT extversion INTO ts_version FROM pg_extension WHERE extname = 'timescaledb';
+  SELECT
+    (string_to_array(extversion,'.'))[1] INTO ts_major
+    FROM pg_extension WHERE extname = 'timescaledb';
+
+  SELECT
+    (string_to_array(extversion,'.'))[2] INTO ts_minor
+    FROM pg_extension WHERE extname = 'timescaledb';
 
   PERFORM add_reorder_policy('policy_test_timestamptz','policy_test_timestamptz_time_idx');
 
   -- some policy API functions got renamed for 2.0 so we need to make
-  -- sure to use the right name for the version
-  IF ts_version < '2.0.0' THEN
```

Comment:
Changed to `ts_major` and `ts_minor` because semantic version compares with a text variable does not really work.